### DISCUSSION TITLE
Cara mengatur letak action button pada tampilan shiny

### DISCUSSION THREAD

**Discussion by MF-Faqih** [teamalgoritma/community#677] [Q&A] [answered]

**Comment by ramdhinafinita** [comment, marked as answer]
Dear Pak @MF-Faqih 
Peletakan action button bisa disesuaikan dengan tampilan yang bapak inginkan, semisal ingin menggunakan action button di page 1 bisa saja dimasukkan kedalam page nya. Atau jika ingin rapih dan page nya tidak terlalu banyak, bisa ditaruh di sidebar.
Sebagai referensi contoh action button bisa dilihat pada link berikut ini: https://shiny.posit.co/r/articles/build/action-buttons/
Terimakasih
